single.php

```diff
@@ -32,17 +32,19 @@
 									else:
 										echo_documents_cover();
 									endif;
+
+									if(get_the_content()):
+										echo "<p>".get_the_content()."</p>";
+									endif;
+
+									if(odm_echo_extras()):
+										odm_echo_extras();
+									endif;
+
+									if(echo_downloaded_documents()):
```

Comment:
Calling the echo_downloaded_documents() function, it will return the p tag event though no content.